File: BaekJoon Online Judge/C++/Step02-If/2753.cpp
//
//  2753.cpp
//  Step2-If
//
//  Created by 김예빈 on 2019. 8. 28..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int year;
    
    cin >> year;
    
    if (year % 400 == 0)
        cout << "1";
    else if (year % 4 == 0 && year % 100 != 0)
        cout << "1";
    else
        cout << "0";
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step02-Conditional Statements/2753.swift
//
//  2753.swift
//  Step02-If
//
//  Created by Yebin Kim on 2019/10/28.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var year = Int(readLine()!)

if let year = year {
    if(year % 4 == 0 && year % 100 != 0) {
        print(1)
    } else if(year % 400 == 0) {
        print(1)
    } else {
        print(0)
    }
}
